makefiles/simulators/Makefile.ius

```diff
@@ -88,6 +88,10 @@ else
    $(error "A valid value (verilog or vhdl) was not provided for TOPLEVEL_LANG=$(TOPLEVEL_LANG)")
 endif
 
+ifeq ($(SIM),ius)
```

Comment:
Is `Makefile.ius` used for anythng else than `SIM=ius`? If not, this conditional can be removed.